components/PostCard.tsx

```diff
@@ -28,21 +58,37 @@ const toBase64 = (str: string) => Buffer.from(str).toString("base64");
 const PostCard = ({ post }: Props) => {
   const dispatch = useAppDispatch();
 
+  const classNameOfPost =
+    isPostADraft(post) === true || isPostInTheFuture(post) === false
+      ? "posts-list-block posts-list-block-draft-or-future"
+      : "posts-list-block";
+
   return (
     <li>
-      <div className="posts-list-block">
+      {isPostADraft(post) ? <DraftPostMark /> : null}
+      {isPostInTheFuture(post) === false ? <FuturePostMark /> : null}
+      <div className={classNameOfPost}>
         <div className="content">
           <Link href={`/post/${post.slug}`}>
             <a className="post-img">
               <Image
-                src={post.featuredImage ?? "/post-images/placeholder.png"}
+                src={
+                  post.featuredImage
+                    ? post.featuredImage
+                    : "/post-images/draft.webp"
+                }
                 alt="blog post image"
                 layout="fill"
                 objectFit="cover"
                 placeholder="blur"
                 blurDataURL={`data:image/svg+xml;base64,${toBase64(
                   shimmer(378, 378)
                 )}`}
+                style={
+                  classNameOfPost.includes("posts-list-block-draft-or-future")
+                    ? { filter: "grayscale(50%)" }
+                    : {}
+                }
```

Comment:
@ntorbinskiy 

There's a better way to check it:

`isPostADraft(post) || isPostInTheFuture`